### PR TITLE
Device motorola-osprey

### PR DESCRIPTION
**Pull request by ata2001**

### CODE REVIEW

File: aports/device/device-motorola-osprey/fb-workaround.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+main() {
+	# Wait untill graphical enviroment is running
```
Comment (ollieparanoid) [inline aports/device/device-motorola-osprey/fb-workaround.sh]: typos: `until`, `environment`

File: aports/device/device-motorola-osprey/90-android-touch-dev.rules [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# udev rules file
+# All device names can be read from weston's logfile (/tmp/weston.log in postmarketOS)
+
+# Touchscreen (use 'weston-calibrator' to calibrate)
+SUBSYSTEM=="input", ATTRS{name}=="synaptics_dsx_i2c", \
+ENV{ID_INPUT}="1", ENV{ID_INPUT_TOUCHSCREEN}="1" \
+ENV{WL_CALIBRATION}="0.954174 0.016888 -2.626083 -0.006031 1.011271 14.175903"
```
Comment (ollieparanoid) [inline aports/device/device-motorola-osprey/90-android-touch-dev.rules]: The calibration is really necessary I guess? Just making sure, we have had success with omitting `ENV{WL_CALIBRATION}` in most cases. See #182 and [the wiki](https://wiki.postmarketos.org/wiki/Screen_Calibration#Weston).

**Comment by ata2001** [comment]
@ollieparanoid Setting the mode in the initramfs didn't help, I still needed to do it after weston started.

**Comment by ata2001** [comment]
@ollieparanoid Fixed typos and removed calibration part from udev rule.

**Comment by ollieparanoid** [comment]
Thanks, shipped! ⛵️

**Pull request merged**